### DISCUSSION TITLE
Trained Model Weights

### DISCUSSION THREAD

**Discussion by Pavan-Naga-207** [Tirth27/Skin-Cancer-Classification-using-Deep-Learning#3] [Q&A] [answered]
bro can you share the saved model please

**Comment by Tirth27** [comment, marked as answer]
We have published our trained weights of the model settings mentioned in the [README.md ](https://github.com/Tirth27/Skin-Cancer-Classification-using-Deep-Learning/blob/main/README.md) in the [Kaggle](https://www.kaggle.com/tirth27/melanoma-classification-kerasonnx-model-weight).
